src/kibana/components/agg_types/controls/ranges.html

```diff
@@ -25,7 +25,9 @@
         name="range.to" />
     </td>
     <td>
-      <button ng-click="agg.params.ranges.splice($index, 1)"
+      <button
+        type="button"
```

Comment:
I really with that this was the fucking default in chrome!! Maybe we should create a directive? Might be a bit heavy...